src/tests/multihost/alltests/test_services.py

```diff
@@ -42,6 +42,10 @@ def test_0002_1736796(self, multihost, localusers):
          cause files domain entries to be qualified, this can break sudo access
         :id: 4b7bdeff-51ba-46ed-b8e1-0685515b87a0
         """
+        # This bug is specific to files provider, so skipping this test
+        # case if SSSD's files provider feature is not available. 
+        if not multihost.client[0].detect_feature:
```

Comment:
Method is called `detect_files_provider()`
And it's a function.

Reply:
We can't call this way, 
if we call getting this error,

-> if not multihost.client[0].detect_feature():
(Pdb) c

>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> PDB continue >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
FAILED                                                                                                                                                                       [100%]
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> traceback >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
**E   TypeError: 'bool' object is not callable**

Reply:
There is **NO** `detect_feature`
You called this method `detect_files_provider()` :
https://github.com/SSSD/sssd/blob/8fd2df7329efb72373c3a9bc06a0320b48e707a3/src/tests/multihost/sssd/testlib/common/qe_class.py#L194

Reply:
yes, updated.

Reply:
It still uses `detect_feature` instead of `detect_files_provider`

Would it be possible to let at least CI run before requesting a review?

I'm not going to review this anymore.